app/controllers/integrated/anything_else_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Integrated
+  class AnythingElseController < FormsController
+    def self.skip?(_application)
```

Comment:
We should just not define this method rather than including it with `false`.

Reply:
fixed! Thanks!